cmd/infracost/run.go

```diff
@@ -68,6 +68,8 @@ func addRunFlags(cmd *cobra.Command) {
 
 	cmd.Flags().StringSlice("exclude-path", nil, "Paths of directories to exclude, glob patterns need quotes")
 	cmd.Flags().Bool("include-all-paths", false, "Set project auto-detection to use all subdirectories in given path")
+	cmd.Flags().String("git-diff", "master", "Show only costs that have git changes compared to the provided branch. Use the name of the current branch to fetch changes from the last two commits")
```

Comment:
It's not clear from the name what this is. I'd consider something like `git-base-ref`. I realise it only supports branches at the moment, but I'm assuming we will support other references later?

Reply:
yes that makes sense, and yes we'd support refs later